src/test/java/stepdefinition/MainPageStep.java

```diff
@@ -0,0 +1,79 @@
+package stepdefinition;
+
+import com.giantshoulder.pageobject.MainPage;
```

Comment:
Please remind me to help you set up a rule for correct import order.

Reply:
Fixed in 90a07093045641fbfedf1a5ba7d45d026dca47f9!